### DISCUSSION TITLE
How can I create PR comment on the file that includes space

### DISCUSSION THREAD

**Discussion by colorbox** [community/community#24485] [Apps, API and Webhooks]
<p>Hi</p>
<p>I want to create PR comment on file that name includes space via GitHub API.</p>
<p>But response is “422 Validation Failed”.</p>
<p>Does anyone know how to create comment on file that includes space in name?</p>
<p>Escaping with backslash, and \u200 was still response 422.</p>
<pre><code>app/models/space_include_\ _file_name.rb
app/models/space_include_\u200_file_name.rb
</code></pre>
<p>I use command and PR below</p>
<pre><code>curl -X POST  -H "Content-Type: application/json" -H "Authorization: token &lt;your token&gt;" --data '{"commit_id":"8ae911bfd368794ba8955793faff475dcf0b81b7","body":"test","path":"app/models/space_include_ _file_name.rb","position":2}' https://api.github.com/repos/colorbox/kata/pulls/3/comments
</code></pre>
<p><a href="https://github.com/colorbox/kata/pull/3/files" rel="nofollow noopener">https://github.com/colorbox/kata/pull/3/files</a></p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/colorbox">@colorbox</a>,</p>
<p>We tried out your cURL command and were able to get it to work. Are you still having this issue?</p>
<p>Thanks!</p>

**Comment by colorbox** [comment]
<p>Thank you so much.</p>
<p>It seems very good.</p>

**Comment by ThirdEyeRose** [comment]
<p>Glad to hear it!</p>
